lola.html

```diff
@@ -0,0 +1,875 @@
+        Presently, in these cases, what should happen is not standardized [I think].  If a Person update happens for 
+        Amy with a “movedTo” value, receiving servers could go looking for any reference including that Actor, and update 
+        their follows.
+      </p>
+
+      <ol>
+        <li>
+          Knowing that Amy’s Activity also moved,  Chris’ server could <b>rewrite</b> his “Like” activity to point to the
+          new location. Presumably if this path is chosen, Chris’ server would do this with all users and all Likes that 
+          pointed to any moved thing.
+        </li>
+        <li>
+          Amy’s Activity could move, and Chris’s server could learn about the move and know that it has Like activities referencing the original article, and nevertheless <b>leave them alone</b>.  They already propagated, their purpose is served.
+        </li>
+      </ol>
+
+      <p>
+        Existing servers seem to follow the “<b>leave them alone</b>” model rather than the <b>“rewrite”</b> model.  The way items propagate initially doesn’t have to define the way they are maintained.  In any case, trying to follow the rewrite model would sure involve some misses, failing back to the leave them alone model.
+      </p>
+
+      <p>
+        For ease of implementation, we lean towards assuming the “leave them alone” model.  Some of the reasons why:
+      </p>
+
+      <ul>
+        <li>How far should change propagation go? If Amy moves her article and Chris’s Like gets updated does that propagate 
+        to Amy’s subscribers? Do they get notified? </li>
+        <li>Where should change propagation go to?  Original follower list at the time the item was sent out? Amy’s follower list today? 
+        </li>
+        <li>A rewrite model, if it were to be relied upon, would require many OTHER servers to participate in a data transfer specification, not just the two servers directly involved.  This increases the difficulty of adoption.
+        </li>
+        <li>
+          Since Activities/Likes can be disconnected in other ways, supporting breadcrumbs seems more useful than trying to achieve perfect rewriting.
+        </li>
+      </ul>
+
+      <p>
+        In addition to assuming that 3rd party servers leave their likes alone in many cases, this specification recommends that the destination server add breadcrumbs to Activity and Like objects to record their prior identities. This is beneficial in showing a nuanced view of older data.  If necessary, a server interested in validating a collection of “Likes” submitted to it, can mostly do so (except for accounts that have disappeared) if the object that the collection of “Likes” it is on maintains a history of past object IDs.  Note that this still ALLOWS other servers to fix up their Likes but they 
+        can do so immediately, later, or never.
+      </p>
+    </section>
+
+
+    <section>
+
+
+      <h2>Discovery </h2>
+
+      <p>
+        TBD - working with Aaron Parecki
```

Comment:
We may want to break this out to a whole separate process. There's one FEP with a very simple OAuth profile, but it seems like the momentum is leaning towards RFC 8414.